src/views/Community/View.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react'
+// import Dashboard from '../Dashboard'
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+import { Row, Progress } from 'reactstrap';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Row' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)